Source/rendering/graphic.h

```diff
@@ -8,26 +8,26 @@
 #include <iostream>
 #include <vector>
 
-#include "../camera/camera_properties.h"
+#include "../camera/camera_property.h"
 
-class Attrib{
-public:
-	void positionAttrib();
-	void colourAttrib();
-	void textureCoordAttrib();
-};
 
-class VertexObjects {
+//Vertex and Attribute Objects
+class VertAttribObject {
 private:
-	unsigned int rVAO, rVBO, rEBO;
+	unsigned int mVAO, mVBO, mEBO;
 
 public:
 	void generateVAO(unsigned int VAO);
 	void generateVBO(const std::vector<float>& vertices, unsigned int VBO);
+	void generateVBO(float vertices[], size_t size, unsigned int VBO);
```

Comment:
Separate calls to `generateVBO` / `EBO` will cause errors/crashes if VAO isn't bound.